src/FedexRest/Services/Pickup/Entity/EmailDetail.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace FedexRest\Services\Pickup\Entity;
+
+class EmailDetail
+{
+    protected string $address;
```

Comment:
As we are dealing with "addresses" in the traditional sense of the word, maybe we could call this `emailAddress` to prevent any confusion.

Reply:
Done